query-node/mappings/src/content/utils.ts

```diff
@@ -219,8 +219,8 @@ export async function processVideoMetadata(
   }
 
   // prepare subtitles if needed
-  if (isSet(meta.subtitles)) {
-    await processVideoSubtitles(ctx, video, assets, meta.subtitles)
+  if (isSet(meta.subtitles) || isSet(meta.clearSubtitles)) {
+    await processVideoSubtitles(ctx, video, assets, meta.subtitles || [], meta.clearSubtitles)
```

Comment:
I think `clearSubtitles` should take priority if defined next to regular subtitles? Maybe we do
```ts
const subtitles = meta.clearSubtitles ? [] : meta.subtitles
if (isSet(subtitles)) {
  // ...
}
```